db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
(The reason it sounds like a polymorphic relationship is that on CustomAttribute, resource_{id,type} _is_ a polymorphic relationship.  Better naming ideas welcome.)